data/spells/lib/spells.lua

```diff
@@ -296,3 +296,57 @@ function Creature:addDamageCondition(target, conditionType, listType, damage, ti
 	target:addCondition(condition)
 	return true
 end
+
+function Creature:addPartyCondition(combat, variant, condition, baseMana)
+	local position = self:getPosition()
+	local party = self:getParty()
+	if not party then
+		self:sendTextMessage(MESSAGE_STATUS_SMALL, "No party members in range.")
+		position:sendMagicEffect(CONST_ME_POFF)
+		return false
+	end
+
+	local members = party:getMembers()
+	members[#members + 1] = party:getLeader()
+
+	if members == nil or type(members) ~= 'table' or #members <= 1 then
```

Comment:
This test is very strange. `getMembers()` will always return a table if `party` is valid, which it is at this point. You shouldn't need to check: if it's nil, because it isn't; it it's a table, because it is; if length is 0 or 1, because it is not (AFAIK a party must have 2 or more players).